capabilities/writetarget/monitoring/pb/platform/write_confirmed.proto

```diff
@@ -0,0 +1,37 @@
+syntax="proto3";
+
+package platform.write_target;
+option go_package = "github.com/smartcontractkit/chainlink-framework/capabilities/writetarget/monitoring/pb/platform;writetarget";
+
+import "execution_context.proto";
+
+// WT confirmed the report was successfully written on-chain
+message WriteConfirmed {
+  string node = 1;
+  string forwarder = 2;
+  string receiver = 3;
+
+  // Report Info
+  uint32 report_id = 4;
+  bytes report_context = 5;
+  bytes report = 6;
+  uint32 signers_num = 7;
+
+  // TODO: Is the report confirmed finalized on-chain, or just observed?
+
+  // When was the report confirmed on-chain
+  string block_hash = 9;
+  string block_height = 10;
+  uint64 block_timestamp = 11;
+
+  // Transmission Info
+  string transmitter = 12;
+  bool success = 13;
+  // TODO: what about EVM's TransmissionInfo parity?
+
+  // [Execution Context]
+  ExecutionContext execution_context = 20;
+
+  // encoder processor info
+  string meta_capability_processor = 21;
```

Comment:
This one is slightly weird in position and structure.
Can you elaborate what is this one about?

Reply:
This is the processor that is being used to decode the product-specific report that is passed from the workflow config to the write target monitor